qemu/tests/cfg/hotplug_mem_share_discard_data.cfg

```diff
@@ -0,0 +1,35 @@
+- hotplug_mem_share_discard_data:
+    only Linux
+    type = hotplug_mem_share_discard_data
+    virt_test_type = qemu
+    login_timeout = 240
+    start_vm = no
+    not_preprocess = yes
+    prealloc_mem = yes
+    slots_mem = 4
+    maxmem_mem = 32G
+    mem_fixed = 4096
+    mem_devs = mem0
+    use_mem_mem0 = no
+    size_mem_mem0 = ${mem_fixed}M
+    guest_numa_nodes = "node0"
+    node_num = 1
+    numa_memdev_node0 = mem-mem0
+    backend_mem = memory-backend-file
+    set_node_hugepage = yes
+    mem-path = /dev/hugepages
+    target_mems = "plug1"
+    share_mem_plug1 = yes
+    size_mem_plug1 = 1G
+    mem-path_plug1 = /dev/hugepages/test_file
```

Comment:
It is only useful for `memory-backend-ram`

Reply:
you mean mem-path_plug1 is only used for `memory-backend-file` right? So then it should be placed under the section  `- backend_file:` right?

Reply:
Right!